requirements-dev.txt

```diff
@@ -16,6 +16,8 @@ mock; python_version < '3.8'
 moto
 mypy
 numpy
+pendulum < 3.0; python_version < '3.12'
+pendulum[test] >= 3.0.0; python_version >= '3.12'
```

Comment:
```suggestion
```

I don't _think_ we need these in this file, since this requirements are pulled in in addition to what we have in `requirements-client.txt`.